spark_ec2.py

```diff
@@ -162,11 +183,74 @@ def setup_external_libs(libs):
 from boto.ec2.blockdevicemapping import BlockDeviceMapping, BlockDeviceType, EBSBlockDeviceType
 from boto import ec2
 
+import yaml
 
 class UsageError(Exception):
     pass
 
 
+def process_conf_file(file_path, parser):
```

Comment:
Yes, I think this is a good time to start creating unit tests.

You're right, this function is a little bit hard to test because it needs a parser object with all the options already filled. Do we want to try only when the paths fails ? Do you have some other tests in mind ?